cookbooks/.gitignore

```diff
@@ -1,3 +1,6 @@
 /apt
 /build-essential
 /cdo-aws
+.kitchen
+*/Berksfile.lock
```

Comment:
the Berksfile.lock is different from Gemfile.lock in that we don't want to check it in?